src/components/SkyBrowser/SkybrowserTabs.jsx

```diff
@@ -0,0 +1,422 @@
+      this.props.api.setPropertyValueSingle(uriTarget, !this.props.isUsingRae);
+    }
+
+    toggleShowSettings() {
+      this.setState({
+          showSettings: !this.state.showSettings
+      })
+    }
+
+    setOpacityOfImage(identifier, opacity) {
+      this.props.skybrowserApi.setOpacityOfImageLayer(this.props.selectedBrowser, Number(identifier), opacity);
+    }
+
+    removeImageSelection(identifier) {
+      this.props.skybrowserApi.removeSelectedImageInBrowser(this.props.selectedBrowser, Number(identifier));
+    }
+
+    createButtons(target) {
+      let skybrowserApi = this.props.skybrowserApi;
+      let targetIsLocked = target.isLocked;
+      const targetId = target.id;
+      const toggleSettings = this.toggleShowSettings;
+
+      const lookButton = { selected : false, icon : "visibility", text : "Look at target", function :  function(targetId){
+         skybrowserApi.lockTarget(targetId);
+         skybrowserApi.adjustCamera(targetId);
+       }};
+      const moveButton = { selected : false,  icon : "filter_center_focus", text : "Move target to center of view", function :  function(targetId){
+         skybrowserApi.centerTargetOnScreen(targetId);
+       }};
+      const lockButton = { selected : targetIsLocked, icon : "lock", text : "Lock aim of target", function :  function(targetId){
+        if(targetIsLocked) {
+          skybrowserApi.unlockTarget(targetId);
+        }
+        else {
+          skybrowserApi.lockTarget(targetId);
+        }
+       }};
+      const showSettingsButton = { selected : this.state.showSettings, icon : "settings", text : "Settings", function : function(targetId){
+         toggleSettings();
+       }};
+
+      const buttons = [lookButton, moveButton, lockButton, showSettingsButton];
+
+      const Buttons = buttons.map((button, index) => {
+        return(
+        <Button
+        onClick={() => {button.function(targetId);}}
+        onMouseLeave={() => this.hideTooltip(index)}
+        className={ button.selected ? styles.tabButtonActive : styles.tabButton}
+        transparent
+        small>
+            <MaterialIcon icon={button.icon} className={"small"} onMouseEnter={() => this.showTooltip(index)}/>
+            { this.state.showButtonInfo[index] &&
+              <TooltipSkybrowser
+                placement="bottom-right"
+                style={this.position}>
+                {button.text}
+              </TooltipSkybrowser>
+            }
+        </Button>
+      );
+    });
+
+      return (
+        <div>
+          <span className={styles.tabButtonContainer} ref={this.setRef('wrapper')}>
+          {Buttons}
+          </span>
+        </div>);
+    }
+
+    createTabs() {
+
+        const { targets, selectedBrowser, skybrowserApi} = this.props;
+        let Buttons = targets[selectedBrowser] ? this.createButtons(targets[selectedBrowser]) : "";
+
+        const allTabs = Object.keys(targets).map((target, index) => {
+
```

Comment:
no extra line here plz

But I'll stop commenting now.The rest seems to mostly be style things that I've already mentioned